src/journal/journald-server.c

```diff
@@ -2928,3 +2973,30 @@ int config_parse_compress(
 
         return 0;
 }
+
+int config_parse_forward_to_socket(
+                const char* unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        SocketAddress* addr = data;
+        int r;
+
+        if (isempty(rvalue))
+                log_syntax(unit, LOG_WARNING, filename, line, 0, "ForwardToSocket= cannot be empty");
```

Comment:
i'd probably just disable the logic again if the empty string is specified. So that people can use drop-ins to turn off the logic again.